qa/tasks/ceph_manager.py

```diff
@@ -1528,6 +1527,18 @@ def tmp(x):
             self.log = tmp
         if self.config is None:
             self.config = dict()
+
+        self.testdir = teuthology.get_testdir(self.ctx)
```

Comment:
this line change should be part of `qa/ceph_manager: minor refactor`

Reply:
Fixed, thanks!